sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -159,3 +162,244 @@ export function toBuffer(input: any): Buffer {
   log.utils("[utils.toBuffer] The converted buffer is: %O.", result);
   return result;
 }
+/**
+ * @internal
+ * Helper function to retrieve active receiver name, if it exists.
+ */
+export function getAssociatedReceiverName(
+  clientEntityContext: ClientEntityContext,
+  sessionId?: string
+): string | undefined {
+  let receiverName: string | undefined;
+  if (sessionId != undefined) {
+    if (clientEntityContext.messageSessions[sessionId]) {
+      receiverName = clientEntityContext.messageSessions[sessionId].name;
+    }
+  } else if (clientEntityContext.batchingReceiver) {
+    receiverName = clientEntityContext.batchingReceiver.name;
+  } else if (clientEntityContext.streamingReceiver) {
+    receiverName = clientEntityContext.streamingReceiver.name;
+  }
+  return receiverName;
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `string` value from given input,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getStringOrUndefined(value: any): string | undefined {
+  if (value == undefined) {
+    return undefined;
+  }
+  return value.toString();
+}
+
+/**
+ *  @ignore
+ * Helper utility to retrieve `integer` value from given string,
+ * or undefined if not passed in.
+ * @param value
+ */
+export function getIntegerOrUndefined(value: any): number | undefined {
+  if (value == undefined) {
+    return undefined;
+  }
+  const result = parseInt(value.toString());
```

Comment:
Which scenario are we trying to catch with the `toString()` here?
I believe we can skip the `undefined` check and the `toString()` conversion and simplify this helper to 
```
const result = parseInt(value);
return result === NaN ? undefined : result;
```

Reply:
The input to parse() functionalities require it to be a string